examples/00_quick_start/sasrec-tf2.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>Other notebooks have a naming pattern: algo_dataset.ipynb, please see: https://github.com/microsoft/recommenders/tree/main/examples/00_quick_start</p><p> </p><p>Would you mind to change this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1530/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>changed to sasrec_amazon.ipynb</p>